src/AmbientSounds.Uwp/Strings/en-US/Resources.resw

```diff
@@ -568,4 +568,10 @@
   <data name="ComputeShader/Octagrams" xml:space="preserve">
     <value>Octagrams</value>
   </data>
+  <data name="RenderingErrorInfo.Message" xml:space="preserve">
+    <value>There was an error that caused the rendering to fail, try restarting the app. If the issue persists, please report it by opening an issue on GitHub.</value>
```

Comment:
It shouldn't really happen, but I figured it'd be best to still inform the user if it does happen instead of having the screensaver just freeze for no apparent reason. I guess it could fail in some niche scenarios such as if the GPU driver crashes/restarts, or if the user is using a device like the Surface Book and happens to detatch the screen while the app is running, as that'd cause the initial GPU device (the GTX1060) to be lost, and that's the one that'd be used.

Reply:
Ok gotcha. Can we change the string to: `There was an issue displaying the screensaver. Try restarting Ambie, or contact us if the issue persists.`

Reply:
Fixed in d2b6fbeee1fda34dc8ae31e67a0e550b17317ab1 👍